programs/marginfi/tests/withdraw_fees.rs

```diff
@@ -0,0 +1,58 @@
+use fixtures::test::{BankMint, TestFixture, TestSettings};
+use marginfi::state::marginfi_group::GroupConfig;
+use solana_program_test::tokio;
+use solana_sdk::pubkey::Pubkey;
+
+#[tokio::test]
+async fn marginfi_group_withdraw_fees_and_insurance() -> anyhow::Result<()> {
+    // Setup test executor with non-admin payer
+    let mut test_f = TestFixture::new(Some(TestSettings::all_banks_payer_not_admin())).await;
+
+    // Mint 1000 USDC to the insurance vault
+    let bank_f = test_f.banks.get(&BankMint::USDC).unwrap();
+    let bank = bank_f.load().await;
+    test_f.usdc_mint.mint_to(&bank.insurance_vault, 1000).await;
+
+    // Create a receiving account and try to withdraw 1000 USDC from the insurance vault
```

Comment:
We could also check that non-authorized accounts don't have the ability to withdraw? For either fees/insurance/etc.

Reply:
That's done line 31 onwards

Reply:
That happens at the end of the test